docs/src/content/docs/getting-started/concepts.mdx

```diff
@@ -29,34 +34,72 @@ Repository
   |-- Directory 2
         |-- Workspace: default
 ```
-
+:::note
+Terrateam executes operations like plan and apply within the context of a Dirspace.
+:::
 ## Changes and File Patterns
 
-When a pull request is opened or updated, Terrateam detects changes in the repository based on the `file_patterns` specified in the [`when_modified`](/configuration-reference/when-modified) configuration. By default, the `file_patterns` are set to `['**/*.tf', '**/*.tfvars']`, which means Terrateam will consider changes to any files with `.tf` or `.tfvars` extensions.
+When a pull request is opened or updated, Terrateam detects changes based on patterns defined in your `.terrateam/config.yml` file under the `when_modified` configuration. The default configuration looks like:
+
+```yaml
+when_modified:
+  file_patterns:
+    - '**/*.tf'
+    - '**/*.tfvars'
+```
 
-Terrateam maps the detected changes to their appropriate [Dirspaces](/getting-started/concepts#dirspace). These mapped Dirspaces are referred to as Changes. When Terrateam executes a Plan or Apply operation on the changes in a pull request, it only considers the Dirspaces that have files modified in the pull request.
+This configuration tells Terrateam to monitor changes to any files with `.tf` or `.tfvars` extensions. Modify these patterns in your configuration file to match your project's structure.
 
-## Auto-Plan and Auto-Apply
+Terrateam maps detected changes to their appropriate [Dirspaces](/getting-started/concepts#dirspace). These mapped Dirspaces are referred to as Changes. When executing operations, Terrateam only considers Dirspaces with modified files in the pull request.
 
-Terrateam offers two automated operations:
+## Auto-Plan and Auto-Apply
```

Comment:
Auto-Plan and Auto-Apply could benefit from a brief summary. Please add a one-line description before the configuration example for clarity.